contracts/identityManager/AlastriaProxy.sol

```diff
@@ -1,23 +1,25 @@
-pragma solidity 0.4.23;
+pragma solidity 0.5.17;
 
 import "../libs/Owned.sol";
 
 contract AlastriaProxy is Owned {
-    address public owner;
+    //address public owner;
```

Comment:
Remove, is not used

Reply:
Este merge request esta repetido, es para cerrarlo, que no tengo acceso para cerrarlo, cierralo please.